frontend/src/utils/grid/cellRenderers.jsx

```diff
@@ -33,35 +33,30 @@ export const LinkRenderer = (props) => {
 export const StatusRenderer = (props) => {
   const location = useLocation()
   return (
-    <Link
-      to={props.node && location.pathname + '/' + props.node.id}
-      style={{ color: '#000' }}
+    <BCBox
```

Comment:
FYI Since status Renderer component is used in below table as well, removing the Link could cause inability to have the redirect link available here, i.e., it offers the users to right click and select "Open Link in New Tab/Window".
![image](https://github.com/user-attachments/assets/161d3a50-3e08-4487-aa02-f3c7b50a9a64)

Reply:
Is this a requirement? Should I fix this?